styles.css

```diff
@@ -6,108 +6,479 @@
 }
 
 body {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
   background: #fff;
   height: auto;
+  border-radius: 15px 50px 30px 5px;
 }
 
 header {
   position: relative;
-  padding: 25px 30px;
+  padding: 25px 0 0 30px;
   margin: 1px 0;
   color: #6070ff;
+  background: #fff;
+  width: 375px;
+  height: 60px;
 }
 
-#main {
-  background: url('./assets/background.png');
+.nav-bar {
+  display: flex;
+  flex-direction: row;
+  width: 375px;
+}
+
+.nav-header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.logo {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  margin-right: 250px;
+}
+
+i {
+  display: flex;
+  flex-direction: row;
+  margin-right: 5px;
```

Comment:
- [ ] Instead of using `display: flex;` multiple times, kindly create a class say
``` css
.flex {
  display: flex;
}
```
and use the class in the HTML element that needs it. This will help your code to follow the DRY rule.

Reply:
I Made A flex class and assigned it to the required elements